lib/galaxy/datatypes/annotation.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import tarfile
+
+from galaxy.datatypes.binary import Binary, CompressedArchive
+from galaxy.datatypes.data import get_file_peek, Text
+from galaxy.util import nice_size
+
+log = logging.getLogger(__name__)
+
+
+class SnapHmm(Text):
```

Comment:
Any `edam_format` or `edam_data` that could be added (also for Augustus)?

Reply:
For snap, I could add http://edamontology.org/data_1364, but for augustus I didn't find anything more specific than http://edamontology.org/data_0950, do you think it's worth adding it anyway?

Reply:
Not an expert, but I suppose it's better than nothing.